test/context.test.js

```diff
@@ -34,17 +36,16 @@ test("no warnings in modern react", () => {
         }
     }
 
-    expect(
-        withConsole(() => {
-            const testRenderer = TestRenderer.create(<App />)
-            expect(testRenderer.toJSON()).toMatchSnapshot()
-
-            TestRenderer.act(() => {
-                box.set(4)
-            })
-            expect(testRenderer.toJSON()).toMatchSnapshot()
-        })
-    ).toEqual({ errors: [], infos: [], warnings: [] })
+    const testRenderer = TestRenderer.create(<App />)
+    expect(testRenderer.toJSON()).toMatchSnapshot()
+
+    TestRenderer.act(() => {
+        box.set(4)
+    })
+    expect(testRenderer.toJSON()).toMatchSnapshot()
+
+    expect(console.warn).not.toHaveBeenCalled()
```

Comment:
Originally it was checking for errors and infos as well. Shouldn't be here as well just to be sure.

Reply:
I agree, thanks.